testsuite/features/support/commonlib.rb

```diff
@@ -176,6 +176,18 @@ def get_client_type(name)
   end
 end
 
+def suse_host?(name)
+  (name.include? 'sle') || (name.include? 'opensuse')
+end
+
+def rh_host?(name)
+  (name.include? 'rhlike') || (name.include? 'centos') || (name.include? 'alma') || (name.include? 'rocky')
+end
+
+def db_host?(name)
```

Comment:
this naming can get confusing as `db` can be seen as database instead of debian - would it be too much of a hassle to change it to
```suggestion
def ddb_host?(name)
```
instead? :grimacing:

Reply:
Thanks, I missed that one when I followed Oscar's suggestion to change `db` into `deb`.